examples/nirf/rfsa-rfsg-NR-analysis-only-modacc.py

```diff
@@ -0,0 +1,467 @@
+        )
+    )
+    pre_trigger_time = response.attr_val
+    number_of_pre_samples = pre_trigger_time * minimum_sample_rate
+    response = rfmxclient.GetAttributeI32(
+        nirfmxinstr_types.GetAttributeI32Request(
+            instrument=rfmxsession,
+            channel_name="",
+            attribute_id=nirfmxinstr_types.NIRFMXINSTR_ATTRIBUTE_RECOMMENDED_NUMBER_OF_RECORDS,
+        )
+    )
+    number_of_records = response.attr_val
+
+    # Configure RFSA Acquisition Settings
+    raise_if_error(
+        rfsaclient.ConfigureNumberOfSamples(
+            nirfsa_types.ConfigureNumberOfSamplesRequest(
+                vi=rfsasession,
+                number_of_samples_is_finite=True,
+                samples_per_record=int(number_of_samples),
+            )
+        )
+    )
+    raise_if_error(
+        rfsaclient.ConfigureIQRate(
+            nirfsa_types.ConfigureIQRateRequest(vi=rfsasession, iq_rate=minimum_sample_rate)
+        )
+    )
+    print(f"Done")
+    print(f"Starting Generation...", end="")
+    raise_if_error(rfsgclient.Initiate(nirfsg_types.InitiateRequest(vi=rfsgsession)))
+    print("Generating")
+    print("Starting Acquisition and Analysis Loop:")
+
+    ###### Start Acquisition and AO Analyze ######
+    read_response = raise_if_error(
+        rfsaclient.Initiate(nirfsa_types.InitiateRequest(vi=rfsasession))
+    )
+    for record in range(number_of_records):
+        read_response = raise_if_error(
+            rfsaclient.FetchIQSingleRecordComplexF32(
+                nirfsa_types.FetchIQSingleRecordComplexF32Request(
+                    vi=rfsasession,
+                    channel_list="",
+                    record_number=record,
+                    number_of_samples=int(number_of_samples),
+                    timeout=10.0,
+                )
+            )
+        )
+
+        iq = [
+            nidevice_grpc.NIComplexNumberF32(real=sample.real, imaginary=sample.imaginary)
+            for sample in read_response.data
+        ]
+
+        ###### Feed IQ Data to Analysis Only ######
+        raise_if_error(
+            rfmxclient.AnalyzeIQ1Waveform(
+                nirfmxnr_types.AnalyzeIQ1WaveformRequest(
+                    instrument=rfmxsession,
+                    selector_string="",
+                    result_name="",
+                    x0=read_response.wfm_info.relative_initial_x,
+                    dx=read_response.wfm_info.x_increment,
+                    iq=iq,
+                    reset=True,
+                )
+            )
+        )
+    ###### Fetch ModAcc results ######
+    modacc_fetch_measurement = raise_if_error(
+        rfmxclient.ModAccFetchCompositeEVM(
+            nirfmxnr_types.ModAccFetchCompositeEVMRequest(
+                instrument=rfmxsession, selector_string="", timeout=10.0
+            )
+        )
+    )
+    print(modacc_fetch_measurement)
+except grpc.RpcError as rpc_error:
+    error_message = rpc_error.details()

```

Comment:
```suggestion
    error_message = str(rpc_error.details() or "")
```
We updated mypy version recently and updated all pre-existing examples on this line to what I suggested to avoid a mypy violation.